apps/landing/src/pages/team.tsx

```diff
@@ -1,8 +1,183 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
-import { ReactComponent as Content } from '~/docs/product/credits.md';
 
+import { ReactComponent as ArrowRight } from '../../../../packages/interface/src/assets/svg/arrow-right.svg';
```

Comment:
Can this be imported from `@sd/interface`?

Reply:
idk if `@sd/interface` aliases to `src/` as its root, but if so we'd have to omit that from the path as well

```suggestion
import { ReactComponent as ArrowRight } from '@sd/interface/src/assets/svg/arrow-right.svg';
```